be/compendium/db/views.py

```diff
@@ -1,3 +1,10 @@
-from django.shortcuts import render
+from rest_framework import generics
 
-# Create your views here.
+from .models import Daily_reports
+from .serializers import DailyReportSerializer
+
+class DailyReportListAPIView(generics.ListAPIView):
+    queryset = Daily_reports.objects.all()
```

Comment:
```suggestion
def get(self, request):
       queryset = Daily_reports.objects.all()
       serializer = DailyReportSerializer(queryset)

       return Response(serializer.data)
```

Reply:
@Panonce do this instead to have more cleaner look.
Note: `Response` is imported from rest_framework